test/resources/theme-ugly.css

```diff
@@ -40,5 +40,10 @@
     --fd-color-accent-7: #a8fac7;
     --fd-color-accent-8: #945ef7;
     --fd-color-accent-9: #e3ba20;
+    --fd-color-accent-10: #5c819d;
+    --fd-color-accent-11: #d17f15;
     --fd-color-accent-12: #41cc50;
+    --fd-color-accent-13: #2b78c5;
+    --fd-color-accent-14: #6367de;
+    --fd-color-accent-15: #984bd2;
```

Comment:
Since we have 15 accent backgrounds now, seems like we should show all 15 on the playground page (instead of just 9).